app/views/shared/_edition_activity_fields.html.erb

```diff
@@ -9,6 +9,10 @@
     <div class="modal-body">
       <%= activity_fields.hidden_field :request_type, value: activity %>
 
+      <% if activity == :skip_review %>
+        <p class="alert alert-warning">You should only skip review in exceptional circumstances only, for example if you're the only person responding to an emergency out of hours call.</p>
```

Comment:
```suggestion
        <p class="alert alert-warning">You should only skip review in exceptional circumstances, for example if you're the only person responding to an emergency out of hours call.</p>
```

I know you didn't write this content, but the two occurrences of "only" were jarring.